sled-agent/src/bootstrap/server.rs

```diff
@@ -138,16 +139,7 @@ impl Inner {
         // TODO-cleanup `sp` is optional because we support running without an
         // SP / any trust quorum mechanisms. Eventually it should be required.
         sp: Option<SpHandle>,
-        // `trust_quorum_members` should be `Some(_)` if we have already
-        // received a sled-agent request from RSS, but will be `None` prior to
-        // RSS completing rack setup. We're unable to verify that the sender of
-        // the sled-agent request is a member of our trust quorum, because it's
-        // that request that defines the trust quorum members. If
-        // `trust_quorum_members` is `None` and we receive a sled-agent request
-        // (and successfully handle it), we'll update
-        // `self.trust_quorum_members` to remember the members sent as part of
-        // that request.
-        trust_quorum_members: Option<Vec<Ed25519Certificate>>,
+        trust_quorum: TrustQuorumMembership,
```

Comment:
You know things are headed in the right direction when large comments can be replaced by types :)